seismic/inventory/add_time_corrections.py

```diff
@@ -0,0 +1,154 @@
+#! /usr/bin/env python
```

Comment:
Create a new Jira ticket to add unit testing for this file. The unit test should verify that
- outfile xml file is same as input xml file except for the addition of csv data lines
- csv data lines recovered from the output xml file match the csv data in the csv input file

Reply:
If you have raised a ticket, please mention the ticket number here.

Reply:
https://gajira.atlassian.net/browse/PV-104